griffon/services/core_queries.py

```diff
@@ -602,7 +588,6 @@ def execute(self, status=None) -> List[Dict[str, Any]]:
                 search_all_upstreams_params["type"] = self.component_type
             if not (self.include_inactive_product_streams):
                 search_all_upstreams_params["active_streams"] = "True"
-            search_all_upstreams_params["released_components"] = "True"
```

Comment:
Note: Why we don't want to filter on released components? Broader search?

Reply:
we need broader search for upstreams